Add documentation about localTs and expectOnly option (#244)
microsoft/dtslint: README.md

- Add to your `package.json` `scripts`: `"dtslint": "dtslint types"`
- `npm run dtslint`

### Options

- `--localTs`

Use your locally installed version of TypeScript.

```sh
dtslint --localTs node_modules/typescript/lib types
```
- `--expectOnly`

> Disable all the lint rules and check for type correctness only.

Reviewer: ```suggestion
Disable all the lint rules except the one that checks for type correctness.
```

Author: Thank you for the suggestion! :D
